packages/lib/server/repository/team.ts

```diff
@@ -175,7 +175,7 @@ export class TeamRepository {
       where: {
         id,
       },
-      select: teamSelect,
+      select: { ...teamSelect, platformBilling: true },
```

Comment:
NIT: I think we should have a separate method to `findByIdIncludePlatformBilling` as I'm sure other places this method is called doesn't require platform billing data.